eslint/babel-eslint-parser/src/index.js

```diff
@@ -12,19 +12,22 @@ import convert from "./convert";
 import analyzeScope from "./analyze-scope";
 import visitorKeys from "./visitor-keys";
 
-let isRunningSupportedVersion;
+let isRunningMinSupportedCoreVersion = null;
 
 function baseParse(code, options) {
-  if (typeof isRunningSupportedVersion !== "boolean") {
-    isRunningSupportedVersion = semver.satisfies(
+  const minSupportedCoreVersion = ">=7.0.0";
```

Comment:
Ok so parse was added first in 7.0?

Reply:
`@babel/core#parse()` was added in `7.0.0-beta.51`, but we need to bump this to `7.2.0` so that we have access to `@babel/parser#tokTypes`.